server/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -24,6 +24,10 @@
 //// SecurityManager impl:
 //// Must have all permissions to properly perform access checks
 
+grant codeBase "${codebase.jna}" {
```

Comment:
Can this go below the "Very speciall jar permissions" comment please, with the rest of the jar specific grants? (yes the SM here is a jar specific grant, but the comment above is specific to secure SM)